exp3/verify.py

```
@@ -0,0 +1,124 @@
+    # 4. 输出最终报告
+    accuracy = (correct_count / total_count * 100) if total_count > 0 else 0
+    duration = time.time() - test_start
+    
+    print("\n" + "="*60)
+    print("测试结果摘要")
+    print("="*60)
+    print(f"总样本数: {total_count}")
+    print(f"正确数量: {correct_count}")
+    print(f"错误数量: {len(errors)}")
+    print(f"平均耗时: {duration/total_count*1000:.1f} ms/个")
+    print(f"---------------------------")
```

Ruff (0.14.5)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)